porting/index.md

```diff
@@ -62,15 +69,54 @@ You can also specify `-j[num]` as it will fetch the files simultaneously. This d
 
 ### Put parts together
 
-Now you need to put all the parts for your device together, and since our tree is based on LineageOS for `halium-7.1` or CyanogenMod for `halium-5.1` you can use the device files they provide.
+Now you need to put all the parts for your device together, and since our tree is based on LineageOS for `halium-7.1` or CyanogenMod for `halium-5.1` you can use the device files they provide. Take care to switch the branch selection in Githubs webpage so you download the correct branches!
 
 Parts that are needed:
-- Device tree
-- Kernel source
+- Device tree (from [LineageOS](https://github.com/LineageOS))
+- Kernel source (also from LineageOS)
 - Vendor tree (e.g. from [TheMuppets](https://github.com/TheMuppets))
 
 You might have to check `lineage.dependencies`/`cm.dependencies` that is included in every LineageOS/CyanogenMod device repo for additional repositories.
 
+### Create a local manifest ###
+
+Create additional entries for the repo tool to download the required parts automatically:
+
+```
+cd <BUILDDIR>/.repo && mkdir local_manifests && cd local_manifests
+gedit <VENDOR>_<CODENAME>.xml. 
+```
+
+The repo tool will accept additional repositories to be synced on top of the ones defined by Halium already. Since you are porting for a new device, you also need to do this work. Later on your local manifest entries can become part of a central repository for device ports.
```

Comment:
The idea is to explain what will happen with that information. We should make it accessible in some form to the public, since people will spend some fair amount of time crafting the local manifest until everything works. We should encourage them to upload it somewhere.